src/amber/controller/render.cr

```diff
@@ -6,7 +6,11 @@ module Amber::Controller
       {% if filename.id.split("/").size > 2 %}
         Kilt.render("{{filename.id}}")
       {% else %}
-        Kilt.render("#{{{path}}}/{{filename.id}}")
+        {% if path == "src/views" %}
+          Kilt.render("{{__FILE__.split('/')[0..-4].join('/').id}}/#{{{path}}}/{{filename.id}}")
```

Comment:
`__FILE__.split('/')[0..-4]` assumes that `__FILE__` is always going to be at the same level. If someone were to render a view from presenter, or even another ecr file it will fail.